feature-styles/HH_HealthDeterminantMeasure.se

```diff
@@ -14,7 +14,7 @@ version="1.1.0" xsi:schemaLocation="http://www.opengis.net/se http://schemas.ope
   <se:FeatureTypeName>hh:EnvHealthDeterminantMeasure</se:FeatureTypeName>
   <se:Rule>
     <se:Description>
-      <se:Title>human health: polygon</se:Title>
+      <se:Title>Human health polygon</se:Title>
```

Comment:
For Renewable and Waste Resource, polygonS and lineS are used in plural - here they are used in singular - maybe align?